### PR TITLE
refactor(utils): drop ie support

### PR DESCRIPTION
**Pull request by sxzz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/utils/util.ts [hunk 1]
```diff
@@ -19,13 +19,6 @@ import { warn } from './error'
 import type { ComponentPublicInstance, CSSProperties, Ref } from 'vue'
 import type { AnyFunction, TimeoutHandle, Hash, Nullable } from './types'
```
Comment (JeremyWuuuuu) [inline packages/utils/util.ts:20]: cc @sxzz Please address these warnings.

Comment (sxzz) [reply packages/utils/util.ts]: Done.

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
@sxzz Whooooop, seems like the PRs I merged before this one is creating conflicts, you might need to address them 🤣

**Comment by sxzz** [comment]
Done.

**Pull request merged**